modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2GetStarted.testcase

```diff
@@ -23,125 +23,196 @@ definition {
 	}
 
 	tearDown {
-		CustomerPortal.customerPortalApisTearDown();
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
 
-		JSONGroup.deleteGroupByName(groupName = "${testSiteName}");
+		CustomerPortal.customerPortalApisTearDown();
 
-		PortalInstances.tearDownCP();
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONGroup.deleteGroupByName(groupName = "${testSiteName}");
+		}
 	}
 
-	@description = "LPS-145260 - Verify if when user completes the on-boarding flow or skips through it, Get Started page doesn't appears again."
+	@description = "Automation ID: LPS-145260 | Test Summary: Verify if when user completes the on-boarding flow or skips through it, Get Started page doesn't appears again"
 	@priority = "4"
 	test GetStartedDontAppearASecondTime {
-		CustomerPortal.showCPObject(title = "Account Flag");
+		task ("Creating objects") {
+			CustomerPortal.showCPObject(title = "Account Flag");
+		}
 
-		CustomerPortal.changeUserRole(
-			accountName = "Test Account 01",
-			accountRoleTitle = "Account Administrator",
-			accountUserName = "Test Test");
+		task ("Change the user role in the project") {
+			Accounts.openAccountsAdmin();
 
-		ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+			CustomerPortal.accountTableProject(title = "Test Account LXC-SM");
 
-		CustomerPortal.enterOnProject(title = "Test Account 01");
+			Accounts.addAssigneeToRoleViaUsersTab(
+				accountRoleTitle = "Account Administrator",
+				accountUserName = "Test Test");
+		}
+
+		task ("Go to CP2.0 and join a project") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
 
-		CustomerPortal.clickOnStartProjectSetup();
+			CustomerPortal.enterOnProject(title = "Test Account LXC-SM");
+		}
 
-		CustomerPortal.clickOnSkipForNowButton();
+		task ("Going through onboardig") {
+			CustomerPortal.clickOnStartProjectSetup();
 
-		CustomerPortal.clickOnSkipForNowButton();
+			CustomerPortal.clickOnSkipForNowButton();
+
+			CustomerPortal.clickOnSkipForNowButton();
```

Comment:
Is there a reason why this is duplicate? Is it correct?

Reply:
Yes, this specific step is when the user is an admin of the project. He has to go through the shipping flow, which is basically 3 modal in a row.